dashboard/app/models/school_info.rb

```diff
@@ -131,14 +132,32 @@ def sync_from_schools
   validate :validate_without_country
   validate :validate_zip
 
+  # Decides whether the school info is complete enough to stop bugging the
+  # teacher for additional information every week.  Different from complete
+  # record validation.
   def complete?
```

Comment:
Is complete used anywhere other than SchoolInfoInterstitialHelper? How sure are we that this change won't break other use cases?

Reply:
As of now, `complete?` is _only_ used in `SchoolInfoInterstitialHelper` - I checked carefully.  Perhaps it makes sense to move this logic into that helper though.